tests/dflowfm/test_tim.py

```diff
@@ -186,27 +220,27 @@ def test_validate_data_for_timeseries_throws_exception_for_incorrect_data(
 
 
 class TestTimParser:
-    triple_data_for_timeseries = {
-        "10": ["1.232", "2.343", "3.454"],
-        "20": ["4.565", "5.676", "6.787"],
-        "30": ["1.5", "2.6", "3.7"],
-    }
+    triple_data_for_timeseries = [
+        {"time": "10", "data": ["1.232", "2.343", "3.454"]},
+        {"time": "20", "data": ["4.565", "5.676", "6.787"]},
+        {"time": "30", "data": ["1.5", "2.6", "3.7"]},
+    ]
 
-    single_data_for_timeseries = {
-        "0.000000": ["0.0000000"],
-        "10.000000": ["0.0100000"],
-        "20.000000": ["0.0000000"],
-        "30.000000": ["-0.0100000"],
-        "40.000000": ["0.0000000"],
-        "50.000000": ["0.0100000"],
-        "60.000000": ["0.0000000"],
-        "70.000000": ["-0.0100000"],
-        "80.000000": ["0.0000000"],
-        "90.000000": ["0.0100000"],
-        "100.000000": ["0.0000000"],
-        "110.000000": ["-0.0100000"],
-        "120.000000": ["0.0000000"],
-    }
+    single_data_for_timeseries = [
```

Comment:
Same comment as above: Do we need this much data?

Reply:
Same reply as above: this is what I got